repairnator/repairnator-core/src/main/java/fr/inria/spirals/repairnator/InputBuildId.java

```diff
@@ -0,0 +1,28 @@
+package fr.inria.spirals.repairnator;
+
+/**
+ * Created by fermadeiral
+ */
+public class InputBuildId {
+
```

Comment:
In order to be completely explicit I propose that you add a public constant int named `NO_PATCH` with a value `-1` for example, and you setup by default patchedBuildId to `NO_PATCH`. Then we'll be able to compare the value of `getPatchedBuildId` against this constant.

Reply:
Done.